clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapChainStorageEngine.java

```diff
@@ -532,14 +532,14 @@ public ReplaceResponse replace(Chain expected, Chain replacement) {
       }
     }
 
-    public ReplaceResponse removeHeader(Chain header) {
+    public boolean removeHeader(Chain header) {
       long suffixHead = chain + OffHeapChainStorageEngine.this.totalChainHeaderSize;
       long prefixTail;
 
       Iterator<Element> iterator = header.iterator();
       do {
         if (!compare(iterator.next(), suffixHead)) {
-          return ReplaceResponse.NO_MATCH;
+          return true;
```

Comment:
No match means replaced?

Reply:
'true' is returned here to avoid evict() call in OffheapChainMap, am just reverting a previous change

Reply:
So true means "do not evict" or something like that

Reply:
yes, please see the usage in OffheapChainMap